### PR TITLE
Allow cancelling tasks/bundles

### PR DESCRIPTION
**Pull request by weshinsley**

### CODE REVIEW

**Review by richfitz — CHANGES_REQUESTED** [review]

File: tests/testthat/test-queue.R [hunk 1]
```diff
@@ -132,7 +132,24 @@ test_that("Submit job and update db", {
...
+  t2$id <- paste0(t2$id, "Z")
+  
+  # Check a list works
+  expect_identical(task_get_ids(c(t1,t2)),
+                   c(t1$id, t2$id))
+  
+  # Check a vector of character ids works
+  expect_identical(task_get_ids(c(t1$id,t2$id)),
+                   c(t1$id, t2$id))
+
```
[the remark this answers is not in the document]
Comment (weshinsley) [reply tests/testthat/test-queue.R:152]: Yeah - bit messy, but I wanted to take advantage of the existing queuer_task object (t). `task_get_ids` is called only by `unsubmit`; I'm testing the inner bit here with a bogus pair of clones.

If I moved it I think I'd need to duplicate the submission machinery from the top of this test, to get a queuer_task object to play with.

[the remark this answers is not in the document]
Comment (weshinsley) [reply tests/testthat/test-queue.R:152]: (I could move 150-151 easily, as that's just testing multiple strings works

[the remark this answers is not in the document]
Comment (weshinsley) [reply tests/testthat/test-queue.R]: Try now... I've duplicated a little bit, but not too much

**Comment by weshinsley** [comment]
I'm not sure why one of the ubuntu CI tests behaves so differently...

**Review by richfitz — APPROVED** [review]

**Pull request merged**